Tests/DependenciesMacrosPluginTests/DependencyClientMacroTests.swift

```diff
@@ -513,22 +513,6 @@ final class DependencyClientMacroTests: BaseTestCase {
         var endpoint: @Sendable () -> Int = { <#Int#> }
       }
       """
-    } expansion: {
-      """
-      struct Client: Sendable {
-        @DependencyEndpoint
-        var endpoint: @Sendable () -> Int = { <#Int#> }
-
-        init(
-          endpoint: @Sendable @escaping () -> Int
-        ) {
-          self.endpoint = endpoint
-        }
-
-        init() {
-        }
-      }
-      """
```

Comment:
this was failing for me and so I had remove it. but not really sure why it doesn't fail on main